### ISSUE TITLE
Erroneous E013 error shown

### ISSUE THREAD

**Issue by XhmikosR** [twbs/bootlint#114]
http://www.bootstrapcdn.com/extras/popular

``` html
<div class="row">
  <div class="col-sm-12">
  ...
```

```
12:14:00.594 "bootlint:" "E013" "Only columns (.col-*-*) may be children of `.row`s" bootlint.min.js:1510
```
